### PR TITLE
feat: Pinata endpoint

### PR DESCRIPTION
**Pull request by rubenkristian**
## Describe your changes
- add pinata module, and endpoint for upload file to pinata

## Issue ticket number / link
- https://blocksphere2020.atlassian.net/browse/DBIO-3080

### CODE REVIEW

File: src/common/secrets/index.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ export const SecretKeyList = {
   ELASTICSEARCH_NODE: process.env.ELASTICSEARCH_NODE || null,
   ELASTICSEARCH_USERNAME: process.env.ELASTICSEARCH_USERNAME || null,
   ELASTICSEARCH_PASSWORD: process.env.ELASTICSEARCH_PASSWORD || null,
+  VUE_APP_PINATA_JWT_KEY: process.env.VUE_APP_PINATA_JWT_KEY || null,
```
Comment (irmannmal) [inline src/common/secrets/index.ts:40]: change to `PINATA_JWT` and please add to https://github.com/debionetwork/debio-backend/blob/main/.env-template

**Review by irmannmal — CHANGES_REQUESTED** [review]

**Comment by irmannmal** [comment]
@rubenkristian please remove https://github.com/debionetwork/debio-backend/blob/main/src/endpoints/authentication/pinata-jwt.model.ts, related module, service and secrets, i think we don't need this recreat `pinata jwt`

**Comment by irmannmal** [comment]
@rubenkristian can u explain this https://github.com/debionetwork/debio-backend/blob/main/notes.ts.txt
if this not used please remove

File: src/common/secrets/index.ts (hunk 1 above)
Comment (rubenkristian) [reply src/common/secrets/index.ts]: done

**Review by irmannmal — APPROVED** [review]

**Pull request merged**